scripts/scil_volume_stats_in_ROI.py

```diff
@@ -39,23 +37,23 @@ def _build_arg_parser():
                    help='Mask volume filename.\nCan be a binary mask or a '
                         'weighted mask.')
 
-    p_metric = p.add_argument_group('Metrics input options')
-    g_metric = p_metric.add_mutually_exclusive_group(required=True)
-    g_metric.add_argument('--metrics_dir',
-                          help='Metrics files directory. Name of the '
-                               'directory containing the metrics files.')
-    g_metric.add_argument('--metrics', dest='metrics_file_list', nargs='+',
-                          help='Metrics nifti filename. List of the names of '
-                               'the metrics file, in nifti format.')
+    g = p.add_argument_group('Metrics input options')
+    gg = g.add_mutually_exclusive_group(required=True)
+    gg.add_argument('--metrics_dir', metavar='dir',
+                    help='Name of the directory containing metrics files: '
+                         'we will \nload all nifti files.')
+    gg.add_argument('--metrics', dest='metrics_file_list', nargs='+',
+                    metavar='file',
+                    help='Metrics nifti filename. List of the names of the '
+                         'metrics file, \nin nifti format.')
 
     p.add_argument('--bin', action='store_true',
-                   help='If set, will consider every value of the mask '
-                        'higher than 0 to be part of the mask, and set to 1 '
+                   help='If set, we will consider every value of the mask '
+                        'higher than 0 \nto be part of the mask '
                         '(equivalent weighting for every voxel).')
-
     p.add_argument('--normalize_weights', action='store_true',
                    help='If set, the weights will be normalized to the [0,1] '
-                        'range.')
+                        'range. \n(We divide by the max,  ')
```

Comment:
`Dividing by the max` ? But this could be removed, as it is exactly what normalizing means in the case of weights. Also, why the `,   `?

Reply:
Oups... sorry, forgot to finish.